Dockerfile

```diff
@@ -1,7 +1,17 @@
-FROM python:3.6
+FROM python:3.6-alpine
 
 ADD . /app
 WORKDIR /app
-RUN pip install -r requirements.txt
-RUN python setup.py install
+
+RUN apk --no-cache add \
+        g++ \
+        make \
+    && pip install -r requirements.txt \
+    && python setup.py install \
+    && apk del \
+        g++ \
+        make
+
+ENV TZ UTC
+
 CMD ["k8s-snapshots"]
```

Comment:
Is running as `root` required for some reason?

Reply:
I don't think so, but need to test. Upstream image runs as root.

Reply:
The command runs as `nobody:nobody` specified in the line 18, am I right?

Reply:
@amatas Yes.